packages/material-ui-icons/package.json

```diff
@@ -21,7 +21,7 @@
   "bugs": {
     "url": "https://github.com/mui-org/material-ui/issues"
   },
-  "homepage": "https://github.com/mui-org/material-ui/tree/master/packages/material-ui-icons",
+  "homepage": "https://material-ui.com/components/icons",
```

Comment:
```suggestion
  "homepage": "https://material-ui.com/components/material-icons",
```

Reply:
The link I had describes the package not just a search page.

Reply:
From my perspective, the current link is about the SvgIcon and the Icon components. Not that much about the material design icons. But fair enough, until we support different set of icons the distinction is fuzy.

Reply:
> From my perspective, the current link is about the SvgIcon and the Icon components.

Then you should propose a change to /components/icons. Because that page starts off with `@material-ui/icons`.